TAScheduler/UnitTests/tests.py

```diff
@@ -1821,33 +1859,38 @@ def setUp(self):
             modality="online",
             credits=3
         )
+        tmpcourse.save()
 
         tmpsection = Section.objects.create(
             section_id=1011,
             course=tmpcourse,
             location="Cool place",
-            meeting_time=datetime.datetime
+            meeting_time=datetime.datetime.now()
         )
+        tmpsection.save()
 
         tmplab = Lab.objects.create(
             section=tmpsection,
         )
+        tmplab.save()
 
         self.lab = LabObj(tmplab)
 
     def test_add_ta(self):
-        self.lab.addTA(TAObj(self.ta))
+        self.lab.addTA(self.ta)
```

Comment:
I actually prefer your old way of sending in the TAObj(), this allows us to use the methods of the TAObj (.hasMaxAssignmets() for example) in our methods so we don't have to redo code ...

Thinking about it again, this might be the same thing: we could send in the TADB item into the lab's method, then just convert it into a TAObj inside the method. (But why would we not just do this in the first place?)

What do you think: should the arg for assigning/removing users be a "class object" or a "database record" (and why)?